### PR TITLE
Add WSL docs

### PR DESCRIPTION
**Pull request by tervay**

### CODE REVIEW

**Comment by tervay** [comment]
I don't entirely understand the Vagrantfile change, those more experienced in Vagrant should review it. Got it from here: https://github.com/hashicorp/vagrant/issues/10576

File: Vagrantfile [hunk 1]
```diff
@@ -72,4 +72,7 @@ Vagrant.configure("2") do |config|
   #  trigger.info = "Pulling datastore from remote container"
   #  trigger.run = {inline: "./ops/dev/pull-datastore.sh pull"}
   #end
+
+
+  config.vm.synced_folder '.', '/vagrant', disabled: true
```
Comment (phil-lopreiato) [inline Vagrantfile:77]: Can you add a comment linking to the GH issue for why we need this?

File: docs/Setup/WSL-Guide.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+After its running (on Windows), it should be added to your WSL path.
+
+### Install Vagrant on WSL
+
+You probably need to install Virtualbox too, as it says in the article.
+
+https://linuxize.com/post/how-to-install-vagrant-on-ubuntu-20-04/
+
+### Edit `Vagrantfile`
```
Comment (phil-lopreiato) [inline docs/Setup/WSL-Guide.md:30]: we no longer need this section now

File: docs/Setup/WSL-Guide.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+https://docs.docker.com/docker-for-windows/wsl/
+
+After its running (on Windows), it should be added to your WSL path.
+
+### Install Vagrant on WSL
+
+You probably need to install Virtualbox too, as it says in the article.
+
+https://linuxize.com/post/how-to-install-vagrant-on-ubuntu-20-04/
```
Comment (phil-lopreiato) [inline docs/Setup/WSL-Guide.md:27]: I'd link to https://www.vagrantup.com/intro/getting-started/install instead

Comment (tervay) [reply docs/Setup/WSL-Guide.md:27]: I linked to that article as it's a way of doing it in the terminal directly; Vagrant's official releases page only provides links to download `zip`s, which devs less experienced with bash may not know how to install onto WSL (eg: I don't)

File: docs/Setup/WSL-Guide.md (hunk 2 above)
Comment (phil-lopreiato) [reply docs/Setup/WSL-Guide.md:30]: can you just delete it instead of commenting it out?

File: docs/Setup/WSL-Guide.md (hunk 3 above)
Comment (phil-lopreiato) [reply docs/Setup/WSL-Guide.md:27]: /shrug okay

**Pull request merged**